v1/examples/schema_example.json

```diff
@@ -0,0 +1,273 @@
+{
+    "nodes": [
+        {
+            "id": 0,
+            "labels": [
+                "Person"
+            ],
+            "count": 2,
+            "properties": [
+                {
+                    "key": "name",
+                    "count": 2,
+                    "types": [
+                        {
+                            "type": "String",
+                            "count": 2
+                        }
+                    ]
+                },
+                {
+                    "key": "age",
+                    "count": 2,
+                    "types": [
+                        {
+                            "type": "Integer",
+                            "count": 2
+                        }
+                    ]
+                },
+                {
+                    "key": "occupation",
+                    "count": 1,
+                    "types": [
+                        {
+                            "type": "String",
+                            "count": 1
+                        },
+                        {
+                            "type": "Null",
+                            "count": 1
+                        }
+                    ]
```

Comment:
Not sure how I missed this, but I would suggest one of the following two cases:

```
A) top `count` being 2 because 1 is String, 1 is Null. `count` being a sum of all types.

B) top `count` stays as 1. And we remove the `Null` from the array of types. So top level `count` only counts non-null values. Total fill percentage can be calculated by property.count vs node.count.
```

Personally I am fine with both.

Reply:
Same applies to edge with `id=3`